run.rb

```diff
@@ -52,27 +52,31 @@
 
 bot.command :stonk, aliases: [:stonks], description: 'get some stonks for tickers' do |event, *args|
```

Comment:
this command is getting huge, should prob just move it entirely to the extension at some point